.pre-commit-config.yaml

```diff
@@ -84,7 +84,7 @@ repos:
         files: ^(homeassistant|pylint)/.+\.py$
       - id: pylint
         name: pylint
-        entry: script/run-in-env.sh pylint -j 0 --ignore-missing-annotations=y
+        entry: script/run-in-env.sh pylint -j 4 --ignore-missing-annotations=y
```

Comment:
You cannot change this in your PR

Reply:
ups, reverted now (changed it only for testing didn't had enough ram)